server/controllers/commentController.js

```diff
@@ -1,30 +1,34 @@
 import articles from '../models/articleModel';
+import func from '../helpers/functions';
 
 const commentController = {
     postComment: (req, res) => {
-        const articleId = parseInt(req.params.id, 10);
-        for (const article of articles) {
-            if (article.id === articleId) {
-                const comment = {
-                    id: article.comments.length + 1,
-                    articleId: parseInt(req.params.id, 10),
-                    createdOn: new Date().toLocaleString(),
-                    // eslint-disable-next-line max-len
-                    articleTitle: article.title,
-                    // eslint-disable-next-line max-len
-                    article: article.article,
-                    authorId: parseInt(req.body.authorId, 10),
-                    comments: req.body.comment,
-                };
+        const articleId = func.toInteger(req.params.id);
+        const article = func.idFinder(articles, articleId);
+        if (article) {
+            const comment = {
+                id: func.idIncrementor(article.comments),
+                articleId: func.toInteger(req.params.id),
+                createdOn: new Date().toLocaleString(),
+                // eslint-disable-next-line max-len
```

Comment:
please remove all Eslint disabling comments and ensure that you do not have Eslint errors.. the essence of Eslint is to enforce the style guide "Airbnb". you should not try to escape it in any way